README.md

```diff
@@ -172,15 +172,15 @@ sudo apt install libicu-dev libtinfo-dev libgmp-dev
 sudo dnf install libicu-devel ncurses-devel
 ```
 
-#### Windows-specific pre-requirements (optional)
+#### Windows-specific pre-requirements
 
-In order to avoid problems with long paths on Windows you can do the following:
+In order to avoid problems with long paths on Windows you can do either one of the following:
 
-1. In the `Local Group Policy Editor`: `Local Computer Policy -> Computer Configuration -> Administrative Templates -> System -> Filesystem` set `Enable Win32 long paths` to `Enabled` (Works
-   only for Windows 10).
+1. Clone the `haskell-ide-engine` to the root of your logical drive (e.g. to
+   `C:\hie`). If this doesn't work try the second option.
```

Comment:
I would note explicitly that you can place it in a parent directory short enough to not trigger the long path issue.
We have take in account that cabal creates a deep path inside `dist-newstyle` to guess a maximum parent path length.
My longest path i've found withouth being exhaustive f.e. is:
`D:\ws\haskell\haskell-ide-engine\dist-newstyle\build\x86_64-windows\ghc-8.6.5\haskell-ide-engine-0.13.0.0\l\hie-test-utils\build\hie-test-utils\lib\HShaskell-ide-engine-0.13.0.0-inplace-hie-test-utils.a`
with 202 chars so i could have the project dir in a longer path (but no much! :grimacing: )
As names could change i would not use a exact number but something like `around x`